packages/migration/migrations/application-config/20230208081755-add-marriage-configuration.js

```diff
@@ -42,7 +45,10 @@ export const down = async (db, client) => {
     await session.withTransaction(async () => {
       await db
         .collection('configs')
-        .updateMany({}, { $unset: { MARRIAGE: '' } })
+        .updateMany(
+          {},
+          { $unset: { MARRIAGE: '', MARRIAGE_REGISTRATION: false } }
```

Comment:
```suggestion
          { $unset: { MARRIAGE: '', MARRIAGE_REGISTRATION: '' } }
```
Although it doesn't matter what we use as the value but using "false" might be a bit confusing.

Reply:
changed the default value to false